lib/blacklight/canvas_course.rb

```diff
@@ -11,9 +12,19 @@ class CanvasCourse
     # A new canvas course accepts the metadata for a course
     # and the pandarus course resourse
     ##
-    def initialize(metadata, course_resource)
+    def initialize(metadata, course_resource, blackboard_export)
       @metadata = metadata
       @course_resource = course_resource
+      @scorm_packages = CanvasCourse.get_scorm_packages(blackboard_export)
+    end
+
+    ##
+    # Extracts scorm packages from a blackboard export zip file
+    ##
+    def self.get_scorm_packages(blackboard_export)
```

Comment:
I think this method ought to belong on `ScormPackage` directly, rather than as a class method on `CanvasCourse`, because (1) it returns an array of `ScormPackage` instances, and (2), it doesn't actually refer to anything that is specific to `CanvasCourse`.